README.md

```diff
@@ -155,7 +155,7 @@ FLAGS:
 OPTIONS:
     -r, --region <bed_file_path>      A bed file that describes the region we want to run the stat
     -s, --stat <stat_type>            The type of statistics we want to perform, by default average. You can specify
-                                      statistic methods: mean, median, hist, percentile=X% (If this is not speficied
+                                      statistic methods: perc_cov, mean, median, hist, percentile=X% (If this is not speficied
```

Comment:
There is also a file `d4tools/src/stat/cli.yml` with this text. I think that controls what is shown in the `d4tools stat --help` text.